docs/training_guides/getting_set_up.rst

```diff
@@ -0,0 +1,172 @@
+
+***************************************
+Step 2: Set Up Your Virtual Environment
+***************************************
+
+Normally, tools like `pip <https://pip.pypa.io/en/stable/>`_ install Python libraries directly to your system. When your Python programs run, they look for the libraries they depend upon in your system. But this can cause problems when different programs need different versions of the same library.
+
+To handle this issue, we recommend you use virtual environments to install Parsons. Virtual environments allow you to install libraries into an isolated environment specific to a project. That way you can use different versions of the same libraries for different projects.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Mac/Linux Virtual Environments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before getting started, check which version of Python you’re running by typing ``python --version`` in your command line. Python 3.4+ includes a virtual environment manager called `venv <https://docs.python.org/3/library/venv.html>`_.  If your version is lower than Python 3.4, you'll have to install a virtual environment manager like `virtualenv <https://virtualenv.pypa.io/en/latest/>`_ if you haven't already. You can do this by typing ``pip install virtualenv`` in the command line.
+
+Next, create a directory to store your virtual environments, for example at the path *home/your_name/virtualenvs*. (Not sure what a path is?  see :ref:`Paths vs $PATHs<path-explainer>`.)
+
+You can use the ``mkdir`` command to create a new directory, ie: ``mkdir home/username/virtualenvs``. We'll refer to the full path to this directory as **$path_to_your_env** below.
+
+The next step is to create your virtual environment within this directory. The commands are different based on whether you're on Python 3.4+ and using venv, or whether you're using an older version with the virtualenv program you just installed.
+
+**If you’ve got Python 3.4 or higher**, type ``python -m venv $path_to_your_env/$your_env_name``. The path should be the directory you created to store the virtual environments, and the environment name is a new name chosen by you.
+
+**If you’ve got a lower Python version**, type ``virtualenv $path_to_your_env/$your_env_name``. Again, the path should be the directory for storing virtual environments, and the env name is a new name.
+
+Regardless of what version you're on, you can activate your virtual environment with the command: ``source $path_to_your_env/$your_env_name/bin/activate``.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Windows Virtual Environments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start by installing virtualenvwrappers from source::
+
+  	git clone git://github.com/davidmarble/virtualenvwrapper-win.git
```

Comment:
We haven't covered Git yet, so should we add a little note above saying `(See below section to install Git)` or something?

Reply:
Yes, great idea! Added.